iTunesUDataPurge/RemoveITunesUCourses.java

```diff
@@ -158,7 +171,17 @@ static public void removeCourses(HashMap<String, String> siteIds, String prefix,
 					
 					// delete course xml
 					//System.out.println(xmlDocument);
-					//wsCall(WS_DELETE_COURSE, uploadURL, xmlDocument, prefix, destination, token);
+					wsCall(WS_DELETE_COURSE, uploadURL, xmlDocument, prefix, destination, token);
```

Comment:
The method wsCall() has a try catch within it that essentially encompasses the entire method with exception to the fir 3 lines. I'm open to suggestions and discussion.